src/scheduler/fifo.c

```diff
@@ -1094,18 +1058,10 @@ main_sched_loop(status *policy, int sd, server_info *sinfo, schd_error **rerr)
 		}
 
 		if (!end_cycle) {
-			if (second_connection != -1) {
-				char *jid = NULL;
-
-				/* get_sched_cmd_noblk() located in file get_4byte.c */
-				if ((get_sched_cmd_noblk(second_connection, &cmd, &jid) == 1) &&
-					(cmd == SCH_SCHEDULE_RESTART_CYCLE)) {
-					log_event(PBSEVENT_SCHED, PBS_EVENTCLASS_JOB, LOG_WARNING,
-						njob->name, "Leaving scheduling cycle as requested by server.");
-					end_cycle = 1;
-				}
-				if (jid != NULL)
-					free(jid);
+			/* TODO: how to handle disconnet from server here */
+			if (has_prio_cmd(sconn)) {
```

Comment:
We can actually rename this function definition as has_prty_cmd.

Reply:
why?  I fix has_prio_cmd more readable.

Reply:
changed to `has_priority_cmd`